alws/utils/modularity.py

```diff
@@ -15,10 +13,6 @@
 from gi.repository import Modulemd
 
 
-def get_random_unique_version():
```

Comment:
I may be wrong, but this func using also for repodata uploader and somewhere else, pls check it